src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -146,5 +148,14 @@ Implementing `dns-prefetch` fallback in the same `<link>` tag causes a bug in Sa
 
 {% endCompare %}
 
+## Effect on Largest Contentful Paint (LCP)
+
+The goal of `dns-prefetch` and `preconnect` is to lessen the amount of time it takes to connect to another origin. The ultimate result is that the time to discover a resource loaded from another origin should be reduced, even if only somewhat.
+
+Where [Largest Contentful Paint (LCP)](/lcp/) is concerned, the least you can do to load a cross-origin resource faster is to use `preconnect`&mdash;but a well-placed [`preload`](/preload-critical-assets/) would most likely be better, as an [LCP candidate](/lcp/#what-elements-are-considered) is a crucial part of the user experience. That said, a `preconnect` can only help, even if not as much as a `preload`.
```

Comment:
A preload image without `fetchpriority="high"` would still not be processed until later. So actually a `preconnect` may be faster. 🤔

Best would be `preload` with `fetchpriority="high"` rather than just `preload`. It's a common misconception that it will cause it to be fetched with high priority just be using the `preload` but that's not the case (and don't think it ever was?). See [here for more info](https://web.dev/priority-hints/#:~:text=Note%20that%20preload%20is%20still%20required%20for%20the%20early%20discovery%20of%20LCP%20images%20included%20as%20CSS%20backgrounds%20and%20can%20be%20combined%20with%20priority%20hints%20by%20including%20the%20fetchpriority%3D%27high%27%20on%20the%20preload%2C%20otherwise%20it%20will%20still%20start%20with%20the%20default%20%22Low%22%20priority%20for%20images), and maybe consider linking to it.

However, I think I would add a line about "for best results serve critical results like LCP resources from the same origin, but where that is not possible, you can at least reduce the impact with `preconnect` or `preload`.

Reply:
And actually, another point is that, with the LCP image discoverable in the initial HTML and a `fetchpriority="high"` you might not even need, nor benefit, from the preload. It’s better for hidden resources.

Reply:
+1 to Barry's last comment. I don't think we want to be recommending preconnect in situations where the LCP resource is discoverable from within the main document. It's really only useful for resources that are not discovered right away, and then the connection can be established in parallel in that discoveraby phrase.

Overall I think the main thing this document is missing as a clear explanation of _when_ preconnect is useful and when it's not (which is true for a lot of these older docs).

As a meta point: In its current state, this document makes it sounds a bit like `preconnect` is one weird trick that will always improve your load performance by 100-500ms.